src/Chainweb/BlockHeader/Genesis.hs

```diff
@@ -91,8 +91,88 @@ genesisParentBlockHash v p = BlockHash $ MerkleLogHash
 -- subsequent block mining can have a `HashTarget` easier (re: higher) than
 -- this. Equivalent to `maxTarget`.
 --
-genesisBlockTarget :: HashTarget
-genesisBlockTarget = maxTarget
+-- When the graph is extended new chains should "enter" with a non-trivial
+-- difficulty in order to avoid races and resulting forks during the first two
+-- or three difficulty adjustement epochs.
+--
+-- On devnet, using maxTarget results in a too high block production and
+-- consecutively orphans and network congestion. The consequence are
+-- osciallations to take serval hundred blocks before the system stabilizes.
+-- This setting cools down initial block production.
+--
+genesisBlockTarget :: ChainwebVersion -> ChainId -> HashTarget
+genesisBlockTarget v@Mainnet01 cid
+    | genesisHeight v cid > 731382 = mainnet20InitialHashTarget
+genesisBlockTarget v@Testnet04 cid
+    | genesisHeight v cid > 278626 = testnet20InitialHashTarget
+genesisBlockTarget Development _ = HashTarget (maxBound `div` 100000)
+genesisBlockTarget _ _ = maxTarget
```

Comment:
We may consider moving this and the following definitions to `Chainweb.Version` (or a submodule of Chainweb.Version`).

Reply:
Add a TODO for this as we really should keep magic block numbers there

Reply:
Also, should this just be `cid > 9` instead of testing for a particular block height threshold? It's the same but would make it ELI5 that this doesn't touch existing blocks.